test/vagrant-service-manager/installer_test.rb

```diff
@@ -29,7 +29,7 @@ module ServiceManager
           @installer.instance_variable_get('@type').must_equal @type
           @installer.instance_variable_get('@machine').must_equal @machine
           @installer.instance_variable_get('@env').must_equal @machine.env
-          @installer.instance_variable_get('@box_version').must_equal 'adb'
+          @installer.instance_variable_get('@options').must_equal({ type: @type }.merge(@options))
```

Comment:
Usually, I would refrain from testing internal state or methods (there are exceptions of cause). Much better to test the external "contract" a class with its public method forms.